--- circ_item_config.js-20110630	2011-06-30 21:54:53.000000000 -0400
+++ circ_item_config.js	2011-06-30 21:55:24.000000000 -0400
@@ -500,7 +500,7 @@
 
 function setMaxFineByCircLocation() {
 	var max_libs = [ 'DTRL', 'SJRLS' ];
-	var mid_libs = [ 'ARL', 'CHRL', 'ECGR', 'FRRLS', 'HCLS', 'NCLS', 'OCRL', 'OHOOP', 'OKRL', 'PMRLS', 'PPL', 'STRL' ];
+	var mid_libs = [ 'ARL', 'CHRL', 'DCPL', 'ECGR', 'FRRLS', 'HCLS', 'NCLS', 'OCRL', 'OHOOP', 'OKRL', 'PMRLS', 'PPL', 'STRL' ];
 
 	var cl = (volume && volume.id != -1) ? volume.owning_lib : currentLocation.id;
 	var max_fine = null;
