--- /openils/var/web/xul/server/index.xhtml.orig	2011-10-05 13:16:50.000000000 -0400
+++ /openils/var/web/xul/server/index.xhtml	2011-10-05 13:20:15.000000000 -0400
@@ -512,16 +512,6 @@
 
           <table class="functionlist">
             <tr>
-              <td height="50"><img src="skin/media/images/portal/helpdesk.png" height="48" width="48"
-              onclick="try { helpdesk(false); } catch(E) { alert(E); }" alt=
-              "Submit A Helpdesk Ticket" /></td>
-
-              <td><img class="newtabimg" src="skin/media/images/portal/newtab2.PNG" alt="Open in a new tab"
-              onclick="try { helpdesk(true); } catch(E) { alert(E); }" /> <a href="#"
-              onclick="try { helpdesk(false); } catch(E) { alert(E); }">&staff.client.portal.helpdesk;</a></td>
-            </tr>
-
-            <tr>
               <td height="50"><img src="skin/media/images/portal/reports.png" height="48" width="48"
               onclick="try { local_admin_reports(false); } catch(E) { alert(E); }" alt=
               "Reports" /></td>
