--- /openils/var/web/opac/skin/default/js/rdetail.js.orig	2011-02-03 14:27:58.000000000 -0500
+++ /openils/var/web/opac/skin/default/js/rdetail.js	2011-02-03 14:28:25.000000000 -0500
@@ -10,7 +10,7 @@
 var rdetailLocalOnly = true;
 var rdetailShowLocal = true;
 var rdetailShowCopyLocation = true;
-var rdetailGoogleBookPreview = true;
+var rdetailGoogleBookPreview = false;
 var rdetailDisplaySerialHoldings = true;
 var rdetailEnableRefWorks = false;
 var rdetailRefWorksHost = 'http://www.refworks.com';
