--- /root/search_form_horiz_overlay.xul	2010-01-07 17:56:34.000000000 -0500
+++ /openils/var/web/xul/server/patron/search_form_horiz_overlay.xul	2011-04-12 13:49:00.000000000 -0400
@@ -56,48 +56,36 @@
         <column id="psc8" flex="1"/>
     </columns>
     <rows id="psr">
-        <row id="psr1">
+        <!-- The Name Row -->
+	<row id="psr1">
             <label id="psl1" control="family_name" 
                 value="&staff.patron_search_form.family_name.label;" 
                 accesskey="&staff.patron_search_form.family_name.accesskey;"/>
             <textbox id="family_name" group="0" context="clipboard"/>
-            <label id="psl7" control="street1" 
-                value="&staff.patron_search_form.street1.label;" 
-                accesskey="&staff.patron_search_form.street1.accesskey;"/>
-            <textbox id="street1" group="1" context="clipboard"/>
-            <label id="psl5" control="phone" 
-                value="&staff.patron_search_form.phone.label;" 
-                accesskey="&staff.patron_search_form.phone.accesskey;"/>
-            <textbox id="phone" group="2" context="clipboard"/>
-            <label id="psl6c" control="card" 
-                value="&staff.patron_search_form.card.label;" 
-                accesskey="&staff.patron_search_form.card.accesskey;"/>
-            <textbox id="card" group="0" context="clipboard"/>
-        </row>
-        <row id="psr2">
             <label id="psl2" control="first_given_name" 
                 value="&staff.patron_search_form.first_given_name.label;" 
                 accesskey="&staff.patron_search_form.first_given_name.accesskey;"/>
             <textbox id="first_given_name" group="0" context="clipboard"/>
-            <label id="psl8" control="street2" 
-                value="&staff.patron_search_form.street2.label;" 
-                accesskey="&staff.patron_search_form.street2.accesskey;"/>
-            <textbox id="street2" group="1" context="clipboard"/>
-            <label id="psl4" control="email" 
-                value="&staff.patron_search_form.email.label;" 
-                accesskey="&staff.patron_search_form.email.accesskey;"/>
-            <textbox id="email" group="0" context="clipboard"/>
-            <label id="psl6b" control="usrname" 
-                value="&staff.patron_search_form.usrname.label;" 
-                accesskey="&staff.patron_search_form.usrname.accesskey;"/>
-            <textbox id="usrname" group="0" context="clipboard"/>
-        </row>
-        <row id="psr3">
             <label id="psl3" control="second_given_name" 
                 value="&staff.patron_search_form.second_given_name.label;" 
                 accesskey="&staff.patron_search_form.second_given_name.accesskey;"/>
             <textbox id="second_given_name" group="0" context="clipboard"/>
-            <label id="psl9" control="city" 
+            <label id="psl6c" control="card" 
+                value="&staff.patron_search_form.card.label;" 
+                accesskey="&staff.patron_search_form.card.accesskey;"/>
+            <textbox id="card" group="0" context="clipboard"/>       
+        </row>
+        <!-- The Address Row -->
+        <row id="psr2">
+            <label id="psl7" control="street1" 
+                value="&staff.patron_search_form.street1.label;" 
+                accesskey="&staff.patron_search_form.street1.accesskey;"/>
+            <textbox id="street1" group="1" context="clipboard"/>
+             <label id="psl8" control="street2" 
+                value="&staff.patron_search_form.street2.label;" 
+                accesskey="&staff.patron_search_form.street2.accesskey;"/>
+            <textbox id="street2" group="1" context="clipboard"/>           
+           <label id="psl9" control="city" 
                 value="&staff.patron_search_form.city.label;" 
                 accesskey="&staff.patron_search_form.city.accesskey;"/>
             <textbox id="city" group="1" context="clipboard"/>
@@ -105,20 +93,36 @@
                 value="&staff.patron_search_form.state.label;" 
                 accesskey="&staff.patron_search_form.state.accesskey;"/>
             <textbox id="state" group="1" context="clipboard"/>
+        </row>
+         <!-- The Phone/Contact Row -->
+        <row id="psr3">
+             <label id="psl11" control="post_code" 
+                value="&staff.patron_search_form.post_code.label;" 
+                accesskey="&staff.patron_search_form.post_code.accesskey;"/>
+            <textbox id="post_code" group="1" context="clipboard"/>           
+            <label id="psl5" control="phone" 
+                value="&staff.patron_search_form.phone.label;" 
+                accesskey="&staff.patron_search_form.phone.accesskey;"/>
+            <textbox id="phone" group="2" context="clipboard"/>
+            <label id="psl4" control="email" 
+                value="&staff.patron_search_form.email.label;" 
+                accesskey="&staff.patron_search_form.email.accesskey;"/>
+            <textbox id="email" group="0" context="clipboard"/>
             <label id="psl6" control="ident" 
                 value="&staff.patron_search_form.ident.label;" 
                 accesskey="&staff.patron_search_form.ident.accesskey;"/>
             <textbox id="ident" group="2" context="clipboard"/>
         </row>
-        <row id="psr14">
+        <!-- The username/alias row -->
+        <row id="psr14"> 
+            <label id="psl6b" control="usrname" 
+                value="&staff.patron_search_form.usrname.label;" 
+                accesskey="&staff.patron_search_form.usrname.accesskey;"/>
+            <textbox id="usrname" group="0" context="clipboard"/>
             <label id="psl14" control="alias" 
                 value="&staff.patron_search_form.alias.label;" 
                 accesskey="&staff.patron_search_form.alias.accesskey;"/>
             <textbox id="alias" group="0" context="clipboard"/>
-            <label id="psl11" control="post_code" 
-                value="&staff.patron_search_form.post_code.label;" 
-                accesskey="&staff.patron_search_form.post_code.accesskey;"/>
-            <textbox id="post_code" group="1" context="clipboard"/>
         </row>
     </rows>
 </grid>
